packages/admin/src/targetGroups/TargetGroupsGrid.tsx

```diff
@@ -263,12 +264,12 @@ export function TargetGroupsGrid({
         },
     ];
 
-    const { filter: gqlFilter, search: gqlSearch } = muiGridFilterToGql(columns, dataGridProps.filterModel);
+    const { search: gqlSearch } = muiGridFilterToGql(columns, dataGridProps.filterModel);
 
     const { data, loading, error } = useQuery<GQLTargetGroupsGridQuery, GQLTargetGroupsGridQueryVariables>(targetGroupsQuery, {
         variables: {
             scope,
-            filter: gqlFilter,
+            filter: { isTestList: { equal: false || null } },
```

Comment:
isn't this:
`filter: { isTestList: { equal: false || null } },`

the same as this:
`filter: { isTestList: { equal: null } },`

Reply:
`isTestList` is not nullable though

Reply:
MainList is not nullable. 
I checked the code again and adapted it, so that isMainList and isTestList are both not nullable.

Reply:
c9a0e639675b628a572cb301f2df744e1cee88d8